agent-maven-3.5/Dockerfile.localdev

```diff
@@ -2,25 +2,33 @@ FROM quay.io/openshift/origin-jenkins-agent-base:v4.0
 
 MAINTAINER Akram Ben Aissi <abenaiss@redhat.com>
 
-ENV MAVEN_VERSION=3.5 \
+ENV MAVEN_VERSION=3.6.3 \
     BASH_ENV=/usr/local/bin/scl_enable \
     ENV=/usr/local/bin/scl_enable \
     PROMPT_COMMAND=". /usr/local/bin/scl_enable" \
     LANG=en_US.UTF-8 \
     LC_ALL=en_US.UTF-8 \
     MAVEN_OPTS="-Duser.home=$HOME"
+    M2_HOME="/opt/maven"
+    MAVEN_HOME="/opt/maven"
+    PATH="${M2_HOME}/bin:${PATH}" 
```

Comment:
`\` are missing at the end of the lines.
The directory containg `mvn` must be added to PATH. It is if we take it from community.
see here https://github.com/openshift/jenkins/pull/1213/files#diff-6fa4f3817d9f36f09234c2a759b521340bd27d92c3f4a56e172330482f5574a3R10

The e2e test uses the community build for testing.